spec/features/state_file/complete_intake_spec.rb

```diff
@@ -590,7 +590,7 @@
         expect(page).to have_text "Enter the code to continue"
         fill_in "Enter the 6-digit code", with: verification_code
         click_on "Verify code"
-        expect(page).to have_text "Code verified! Authentication needed to continue."
+        expect(page).to have_text "We found your account! Please confirm your identity to finish signing in."
```

Comment:
<nit> I'd move these last few bits over to i18n as well in case we change them again